features/networking/ovn.feature

```diff
@@ -0,0 +1,25 @@
+Feature: OVN related networking scenarios
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-32205
+  @admin
+  Scenario: Thrashing ovnkube master IPAM allocator by creating and deleting various pods on a specific node
+    Given the env is using "OVNKubernetes" networkType
+    And I store all worker nodes to the :nodes clipboard
+    And I have a project
+    Given I obtain test data file "networking/generic_test_pod_with_replica.yaml"
+    When I run the steps 10 times:
+    """
+    When I run oc create over "generic_test_pod_with_replica.yaml" replacing paths:
+      | ["spec"]["replicas"]                     | 25                      |
+      | ["spec"]["template"]["spec"]["nodeName"] | <%= cb.nodes[0].name %> |
+    Then the step should succeed
+    And 25 pods become ready with labels:
```

Comment:
not sure all platform can schedule 25 pods in one node at same times and make them running.  some of them may be 'pending' if resource is not enough.

Reply:
I guess we can mark this test as low priority, not to be run everytime. 25*10 (250) is chosen to check IP address recycling by IPAM on a node.

Reply:
@anuragthehatter look like the  the scenario is Thrashing ovnkube master IPAM allocator, after pods getting 10 ip addresses each time, do we have any way to check if those ip addressed are from ovnkube master IPAM allocator?

For SDN, we can check pods' ip addresses allocated from CNI IPAM:
sh-4.2# ls -l /var/lib/cni/networks/openshift-sdn
total 44
-rw-r--r--. 1 root root 64 Jul  9 14:00 10.130.2.10
-rw-r--r--. 1 root root 64 Jul  9 14:00 10.130.2.11
-rw-r--r--. 1 root root 64 Jul  9 13:48 10.130.2.2
-rw-r--r--. 1 root root 64 Jul  9 13:48 10.130.2.3
-rw-r--r--. 1 root root 64 Jul  9 16:15 10.130.2.78
-rw-r--r--. 1 root root 64 Jul  9 16:15 10.130.2.79
-rw-r--r--. 1 root root 64 Jul  9 16:15 10.130.2.80
-rw-r--r--. 1 root root 64 Jul  9 16:15 10.130.2.81
-rw-r--r--. 1 root root 64 Jul  9 16:15 10.130.2.82
-rw-r--r--. 1 root root 64 Jul  9 16:15 10.130.2.83
-rw-r--r--. 1 root root 11 Jul  9 16:15 last_reserved_ip.0
sh-4.2#

Reply:
let me check with dev. It seems black box to me for OVN but not sure at the moment

Reply:
@weliang1 Confirmed with Aniket, it seems internal to OVN northd and there is no way to populate IPs like above in OpenshiftSDN. So seems black box for now. Yea, this test just make sure that any pod should not stuck in pending state with IP allocation error while ongoing thrashing activity

Reply:
@anuragthehatter Thanks for confirming this. Better to have a way to check allocated IP as SDN does in the future.
/LGTM

Reply:
Confirmed with dev. Even 5 pods creation deletion would be sufficient to check IPAM allocator resiliency. So we will now only create 5 pods but in loop of 10. So it will be only 5 pods at a time. Should be a good functional test to keep. cc @akostadinov